### ISSUE TITLE
May I ask what the version of node is?

### ISSUE THREAD

**Issue by chaoyang915506** [tebben/cesium-gaussian-splatting#2]
May I ask what the version of node is? I found that I started a new project by following the code of your project and found that the effect was not realized. Finally, I used your dependency to see the effect.

**Comment by chaoyang915506** [comment]
How can I be compatible with @mkkellogg/gaussian-splats-3d version 0.4.3 or higher

**Comment by tebben** [comment]
I use `nvm` to manage node version and am currently on v23.1.0.

```sh
nvm install v23.1.0
nvm alias default v23.1.0
```

For me the following works

```sh
# clone repo
git clone git@github.com:tebben/cesium-gaussian-splatting.git

# open project with vscode
code ./cesium-gaussian-splatting

# install deps
npm install

# run
npm run dev
```

Open [viewer](http://localhost:3001/cesium-gaussian-splatting/)

**Comment by tebben** [comment]
I just tested with the latest version of @mkkellogg/gaussian-splats-3d version 0.4.5 and I cannot see the splat anymore. Related issue: https://github.com/mkkellogg/GaussianSplats3D/issues/345

version 0.4.3 should work fine

**Comment by chaoyang915506** [comment]
Okay, thank you very much. That was very helpful

**Comment by Fregettia** [comment]
https://github.com/mkkellogg/GaussianSplats3D/issues/357#issuecomment-2675047376

The [dev](https://github.com/mkkellogg/GaussianSplats3D/tree/dev) branch of gaussian-splats-3d seems to fix the splat displaying issue from 0.4.4.
